interface/wx/grid.h

```diff
@@ -2511,9 +2511,6 @@ class wxGridTableBase : public wxObject
     /**
         Delete rows from the table.
 
-        Notice that currently deleting a row intersecting a multi-cell (see
```

Comment:
Do you think it could be useful to describe what happens with the multicells here? Or maybe add a separate section about multi-cells explaining this and what happens when inserting too?

Reply:
The name was trying to refer to the original grid (before edits) rendered as a string, with (to be) insertions/deletions mentioned in the side-lines.
I use `ori`(ginal) myself and forgot to change this after seeing wx had `orig` usages. That's still ambiguous though (e.g. in wx `orig[inal]Width` vs. `orig[in]ScaleX`) so maybe just something with `original` or `before`. `old` could also work but that seemed less precise.

I'll drop the `s` prefix. I use it (well, not for static strings...) or `str` sometimes only for strings in usually short-lived code when the same thing is represented by different types. E.g. `width` as integer and string, where I dislike alternatives such as `widthStr`, `widthAsStr`, or `s_width` even more. Though here I suppose it can already be more clear that it's a string from the postfix which is `Annotated` right now, I'd still be interested to know if there's some preference in wx for such cases.

Reply:
I agree that "multicell" is slang and it would probably be better to use something more formal in the manual. "merged cells" sounds good to me, but the most consistent with the names actually used in the code would be "cells spanning multiple rows/columns" which is, of course, also very long. So I basically don't know what would be the best here and so would like to leave the choice to you.

Reply:
Merged cells came from (not only) LibreOffice. I'll also mention the other longer term along with, probably, "merged cells".